regression/goto-instrument/restrict-function-pointer-to-complex-expression/test.desc

```diff
@@ -0,0 +1,7 @@
+CORE
+test.c
+--restrict-function-pointer "use_fg.function_pointer_call.1/f,g"
```

Comment:
:question: What is the point of `function_pointer_call`, why isn't the syntax just `use_fg.1/f,g`?
:question: Can I specify more all calls? e.g. `use_fg/f,g` for all calls inside `use_fg`?

Reply:
1. This could be shortened, yes.
2. No, but something similar is in a follow up PR

Reply:
Will defer 1 to the next PR.